suit_generator/cmd_payload_extract.py

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+"""CMD_PAYLOAD_EXTRACT CLI command entry point."""
+
+import cbor2
+import logging
+
+log = logging.getLogger(__name__)
+
+PAYLOAD_EXTRACT_CMD = "payload_extract"
+
+def add_arguments(parser):
+    """Add additional arguments to the passed parser."""
+    cmd_payload_extract_arg_parser = parser.add_parser(PAYLOAD_EXTRACT_CMD, help="Create raw cache structure.")
+
+    cmd_payload_extract_arg_parser.add_argument(
+        "--input-envelope",
+        required=True,
+        help="Input envelope file path."
+    )
+    cmd_payload_extract_arg_parser.add_argument(
+        "--output-envelope",
+        required=True,
+        help="Output envelope file path."
+    )
+    cmd_payload_extract_arg_parser.add_argument(
+        "--payload-name",
+        required=True,
+        help="Name of the integrated payload to extract."
+    )
+    cmd_payload_extract_arg_parser.add_argument("--output-payload-file", required=False, help="Output payload file path to store the extracted payload."
+                                                + "If not provided, the payload will not be stored to a file.")
+
+    cmd_payload_extract_arg_parser.add_argument(
+        "--payload-replace-name",
```

Comment:
should not that be `path` not `name`?
I find it difficult to understand this arg description.

Reply:
You're absolutely right, should be path, changed this